python-debian 0.1.39 does not work on Windows

Example CI run: https://github.com/fsfe/reuse-tool/runs/1778624975

```
py.test
ImportError while loading conftest 'D:\a\reuse-tool\reuse-tool\tests\conftest.py'.
tests\conftest.py:23: in <module>
    from debian.copyright import Copyright
c:\hostedtoolcache\windows\python\3.6.8\x64\lib\site-packages\debian\copyright.py:59: in <module>
    from debian import deb822
c:\hostedtoolcache\windows\python\3.6.8\x64\lib\site-packages\debian\deb822.py:321: in <module>
    import debian.changelog
c:\hostedtoolcache\windows\python\3.6.8\x64\lib\site-packages\debian\changelog.py:93: in <module>
    import pwd
E   ModuleNotFoundError: No module named 'pwd'
mingw32-make: *** [Makefile:69: test] Error 4
Error: Process completed with exit code 1.
```

python-debian added a dependency on `pwd`, which only exists on Unix-y systems.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -64,8 +64,8 @@ The versions follow [semantic versioning](https://semver.org).
 
 - Fixed a regression where unused licenses were not at all detected. (#285)
 
-- Declared dependency on `python-debian <= 0.1.38`. Later versions of the
-  dependency do not import on Windows. (#310)
+- Declared dependency on `python-debian != 0.1.39` on Windows. This version does
+  not import on Windows. (#310)
 
 - `MANIFEST.in` is now recognised instead of the incorrect `Manifest.in` by
   `addheader`. (#306)

--- a/setup.py
+++ b/setup.py
@@ -5,6 +5,7 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 import glob
+import platform
 import shutil
 import subprocess
 from distutils import cmd
@@ -15,8 +16,10 @@ from setuptools import setup
 from setuptools.command.build_py import build_py
 
 requirements = [
-    # For parsing .reuse/dep5. TODO: Later versions do not work on Windows.
-    "python-debian <= 0.1.38",
+    # For parsing .reuse/dep5.
+    "python-debian"
+    if platform.system() != "Windows"
+    else "python-debian != 0.1.39",
     # For downloading from spdx/spdx-license-list-data. Could maybe use
     # standard library instead?
     "requests",

